src/share/poudriere/ports.sh

```diff
@@ -188,10 +188,14 @@ check_portsnap_interactive() {
 
 if [ ${CREATE} -eq 1 ]; then
 	# test if it already exists
-	porttree_exists ${PTNAME} && err 2 "The ports tree, ${PTNAME}, already exists"
-	maybe_run_queued "${saved_argv}"
 	: ${PTMNT="${BASEFS:=/usr/local${ZROOTFS}}/ports/${PTNAME}"}
 	: ${PTFS="${ZPOOL}${ZROOTFS}/ports/${PTNAME}"}
+	porttree_exists ${PTNAME}
+	if [ $? -eq 0 ]; then
```

Comment:
Granted we don't set `set -e` for the `ports` command, we may in the future so it would be better to test the call directly.